lib/main.js

```diff
@@ -15,6 +15,7 @@ Object.assign(exports, {
 
     // options.loadReason can be install/enable/startup/upgrade/downgrade
     PlacesProvider.links.init();
+    options.telemetry = true;
```

Comment:
Nit: Can you please turn it off by default?
We don't have means to send the data yet, which means they will just accumulate in simple storage.
Let's turn it on by default once we can send the data.

This code is still useful because it is probably foundational for all other telemetry.